src/System.Management.Automation/engine/SessionState.cs

```diff
@@ -337,10 +337,8 @@ internal void InitializeFixedVariables()
             this.GlobalScope.SetVariable(v.Name, v, asValue: false, force: true, this, CommandOrigin.Internal, fastPath: true);
 
             // $PID
-            Process currentProcess = Process.GetCurrentProcess();
-            v = new PSVariable(
-                    SpecialVariables.PID,
-                    currentProcess.Id,
+            v = new PSVariable(SpecialVariables.PID,
+                    Environment.ProcessId,
```

Comment:
```suggestion
            v = new PSVariable(
                    SpecialVariables.PID,
                    Environment.ProcessId,
```

Reply:
Why ?  $PID definition is now the only different style from the cs file.
It's not very important but I just want to understand the reason behind it and or if you just made a mistake.
![image](https://github.com/user-attachments/assets/b2d22c43-6568-4b33-afa0-bdd1054dc39f)

Reply:
I remember CodeFactor reported https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1117.md previously.

Reply:
Understood. Thank you.